src/libcore/option.rs

```diff
@@ -233,10 +233,10 @@ impl<T> Option<T> {
     /// [`usize`]: ../../std/primitive.usize.html
     ///
     /// ```
-    /// let num_as_str: Option<String> = Some("10".to_string());
+    /// let num_as_str: Option<String> = Some("12".to_string());
```

Comment:
To address the problem more directly it might make sense to change this completely -- the fact that the string contents are numbers are irrelevant. Maybe
```rust
let text: Option<String> = Some("Hello, World!".to_string());
let text_length: Option<usize> = text.as_ref().map(|t| t.len());
```

Reply:
That's a good idea. I'll change that.